events/urls.py

```diff
@@ -3,9 +3,8 @@
 from . import views
 from django.views.decorators.csrf import csrf_exempt
 
-
 urlpatterns = [
-    path('list/<int:year>/',views.get_events,name="get_events"),
+    path('list/<year>/',views.EventView.as_view(),name="events_list"),
```

Comment:
```suggestion
    path('list/<int:year>/',views.EventView.as_view(),name="events_list"),
```
Since years are always integral, it makes sense to retain `int` here.